doc-content/enterprise-only/openshift/offline-settings-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='offline-settings-proc']
+= Mounting the custom settings file to access your Maven mirror repository
+
+If your OpenShift environment does not have a connection to the public Internet, you must mount the custom settings file that you prepared according to <<offline-repo-proc>> in your {KIE_SERVER} deployment.
+
+ifeval::["{context}"=="openshift-immutable"]
+This procedure does not apply to source to image (S2I) {KIE_SERVER} deployment.
+endif::[]
+
+. In a command line window, change to the directory where the custom `settings.xml` file is located.
+
+. Create an OpenShift configuration map from the file. Run the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap maven_settings --from-file=settings.xml=settings.xml
```

Comment:
The command is `oc create configmap settings.xml -n NAMESPACE --from-file jboss-settings.xml` where NAMESPACE is the namespace/project where your template will be deployed, jboss-settings.xml is the modified settings.xml file, file name from download link in chapter 2.5 is jboss-settings.xml